droid-hal-device.inc

```diff
@@ -603,6 +603,8 @@ fi
 # Droid config droppings
 /*.rc
 /default.prop
+/selinux_version
+/selinux_contexts
```

Comment:
These will break other devices who don't have these files. Please do this manually for now.
HADK is moving to modular layout where such things are resolved (PDF will be updated soon).
You need to migrate to modular at some point (hybris-11.0 is the last version to support non-modular layout), it's not difficult: http://piratepad.net/dhd2modular